### PR TITLE
Enable checkpoints saving and training resuming

### LINKED ISSUE #664: Enable checkpoints saving and training resuming

### PR DESCRIPTION
**Pull request by kuke**
Resolve #664

### CODE REVIEW

File: fluid/DeepASR/tools/profile.py [hunk 1]
```diff
@@ -125,7 +125,8 @@ def profile(args):
         class_num=1749,
         parallel=args.parallel)
 
-    adam_optimizer = fluid.optimizer.Adam(learning_rate=args.learning_rate)
+    adam_optimizer = fluid.optimizer.Momentum(
```
Comment (pkuyym) [inline fluid/DeepASR/tools/profile.py]: Please change `adam_optimizer` to `momentum_optimizer`

Comment (kuke) [reply fluid/DeepASR/tools/profile.py]: Done. Thx!

Comment (kuke) [reply fluid/DeepASR/tools/profile.py]: Done. Thanks!

**Review by pkuyym — APPROVED** [review]
LGTM

**Pull request merged**